src/components/VideoCard/VideoCard.Styled.jsx

```diff
@@ -0,0 +1,32 @@
+import styled from 'styled-components';
+
+export const VideoCardContainer = styled.div`
+  width: 345px;
+  height: 345px;
```

Comment:
I suggest you use relative units like rem. This helps you achieve a more responsive application. Ideally, you should only use pixels for font sizes, borders, and small stuff.

https://thecssworkshop.com/lessons/relative-units